components/ingest-service/backend/elastic/jobs.go

```diff
@@ -175,9 +176,9 @@ func (es *Backend) DeleteMarkedNodes(ctx context.Context, threshold string) (upd
 	}
 
 	// Clear Scroll
-	scrollService.Clear(ctx)
+	scrollService.Clear(ctx) // nolint: errcheck
 
-	return
+	return // nolint:nakedret
```

Comment:
This function and some of the others are maybe worth reviewing.  Because we shadow `err` in the loop body, it is pretty hard to understand what error conditions here actually result in an error being returned.